easybuild/easyblocks/generic/bundle.py

```diff
@@ -124,8 +125,22 @@ def __init__(self, *args, **kwargs):
                 comp_cfg['name'] = comp_name
                 comp_cfg['version'] = comp_version
 
+                # The copy above may include unexpected settings for common values.
+                # In particular for a Pythonbundle we have seen a component inheriting
+                #  runtest = True
+                # which is not a valid value for many easyblocks.
+                # Reset runtest to the original default, if people want the test step
+                # they can set it explicitly (or it can be set by the component easyblock).
```

Comment:
Shall we mention default_component_specs for replacing `runtest = True` at the toplevel?

```suggestion
                # they can set it explicitly, in default_component_specs or by the component easyblock
```